src/app/owners/owner-list/owner-list.component.ts

```diff
@@ -51,4 +53,25 @@ export class OwnerListComponent implements OnInit {
     this.router.navigate(['/owners/add']);
   }
 
+  searchByLastName(lastName: string)
+  {
+      console.log("inside search by last name starting with "+(lastName));
+      this.ownerService.searchOwners(lastName)
+      .subscribe(
+      (owners)=>{
+
+       this.owners = owners;
+       document.getElementById("ownersTable").style.visibility="visible";
```

Comment:
Not sure that accessing to the DOM for setting CSS is the best way

Reply:
Can you please  advice  any other way.

Reply:
Maybe look at the `[hidden]` attribute or the `*ngIf`directive?